board/xilinx/zynq/board.c

```diff
@@ -44,7 +44,7 @@ static xilinx_desc fpga100 = XILINX_XC7Z100_DESC(0x100);
 #ifndef CONFIG_TPL_BUILD
 static struct {
   uint32_t hardware;
-  uint32_t serial_number;
+  char uuid[17];
```

Comment:
Also, is the mfg serial number not going to be in this factory params struct anymore?